README.md

```diff
@@ -28,6 +28,10 @@ Dietstory requires JDK7 with [JCE Unlimited Strength Policy 7](https://www.oracl
 
 To build the project, [Apache Ant](https://ant.apache.org/) 1.9+ cli tool is required. Alternatively, installing [NetBeans IDE](https://netbeans.apache.org/) provides the necessary tools to build without explicitly running a terminal. If you are running NetBeans, ensure to add the external jars under `cores\` to the IDE compile-time build settings.
 
+#### Update
```

Comment:
Probably better to let people know about the Apache Ivy dependency. Also `Apache Ant version 1.9.14` should be the one that people use.

Reply:
specified general Ant 1.9 versions.